packages/vuetify/src/components/VSheet/VSheet.sass

```diff
@@ -3,7 +3,7 @@
 
 // Theme
 +theme(v-sheet) using ($material)
-  background-color: map-get($material, 'cards')
+  background-color: map-get($material, 'cards') !important
```

Comment:
What is the reason for making this !important? `v-sheet` is used by a ton of components, this is a recipe for disaster.